website/docs/addon_maya_artist.md

```diff
@@ -297,11 +297,22 @@ Then I've put everything into `arm_rig` group.
 When you've prepared your hierarchy, it's time to create *Rig instance* in AYON.
 Select your whole rig hierarchy and go **AYON → Create...**. Select **Rig**.
 Set is created in your scene to mark rig parts for export. Notice that it has
-two products - `controls_SET` and `out_SET`. Put your controls into `controls_SET`
+four products - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
```

Comment:
```suggestion
four children object sets - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
```
Best to avoid confusion here with AYON products?